### ISSUE TITLE
dr_wav: dead code in drwav_write_pcm_frames_be

### ISSUE THREAD

**Issue by krcroft** [mackron/dr_libs#81]
```
3081 drwav_uint64 drwav_write_pcm_frames_be(drwav* pWav, drwav_uint64 framesToWrite, const void* pData)
3082{
3083    drwav_uint64 bytesToWrite;
3084    drwav_uint64 bytesWritten;
3085    drwav_uint32 bytesPerSample;
3086    const drwav_uint8* pRunningData;
3087
3088    if (pWav == NULL || framesToWrite == 0 || pData == NULL) {
3089        return 0;
3090    }
3091
3092    bytesToWrite = ((framesToWrite * pWav->channels * pWav->bitsPerSample) / 8);
    	cond_at_most: Condition bytesToWrite > 18446744073709551615UL, taking false branch. Now the value of bytesToWrite is at most 18446744073709551615.
3093    if (bytesToWrite > DRWAV_SIZE_MAX) {
3094        return 0;
3095    }
3096
3097    bytesWritten = 0;
3098    pRunningData = (const drwav_uint8*)pData;
3099
3100    bytesPerSample = drwav_get_bytes_per_pcm_frame(pWav) / pWav->channels;
3101    
    	cond_between: Condition bytesToWrite > 0UL, taking true branch. Now the value of bytesToWrite is between 1 and 18446744073709551615.
    	cond_between: Condition bytesToWrite > 0UL, taking true branch. Now the value of bytesToWrite is between 1 and 18446744073709551614.
3102    while (bytesToWrite > 0) {
3103        drwav_uint8 temp[4096];
3104        drwav_uint32 sampleCount;
3105        size_t bytesJustWritten;
3106        drwav_uint64 bytesToWriteThisIteration;
3107
    	assignment: Assigning: bytesToWriteThisIteration = bytesToWrite.
3108        bytesToWriteThisIteration = bytesToWrite;
    	cond_between: Condition bytesToWriteThisIteration > 18446744073709551615UL, taking false branch. Now the value of bytesToWriteThisIteration is between 1 and 18446744073709551614.
    	cond_between: Condition bytesToWriteThisIteration > 18446744073709551615UL, taking false branch. Now the value of bytesToWriteThisIteration is between 1 and 18446744073709551615.
    	between: At condition bytesToWriteThisIteration > 18446744073709551615UL, the value of bytesToWriteThisIteration must be between 1 and 18446744073709551615.
    	dead_error_condition: The condition bytesToWriteThisIteration > 18446744073709551615UL cannot be true.
3109        if (bytesToWriteThisIteration > DRWAV_SIZE_MAX) {
    	
CID 277358 (#1 of 1): Logically dead code (DEADCODE)
dead_error_line: Execution cannot reach this statement: bytesToWriteThisIteration =....
3110            bytesToWriteThisIteration = DRWAV_SIZE_MAX;
3111        }
```

**Comment by mackron** [comment]
This is only a minor issue. Fixed in 8d7d5cfcd3b858c3e1f4d18afad09afb52622f81.

**Comment by krcroft** [comment]
Made quick work of those @mackron :rocket:; the three updated codecs are in the analysis queue - we should see results in the next couple minutes.
